playbooks/vars/defaults.yaml

```
@@ -48,3 +48,6 @@ testconfig_public_key: ~/.ssh/id_rsa.pub
 cirros_url: http://download.cirros-cloud.net/0.5.1/cirros-0.5.1-x86_64-disk.img
 rhcos_meta_url: https://raw.githubusercontent.com/openshift/installer/master/data/data/rhcos.json
 # Define rhcos_url to override use of rhcos_meta_url
+
+enabled_services:
+  - /usr/share/openstack-tripleo-heat-templates/environments/services/octavia.yaml
```

If implementing the above suggestion, the `enabled_services` entries would look like:
```suggestion
  - octavia
```